gapis/api/vulkan/frame_loop.go

```diff
@@ -1479,6 +1490,13 @@ func (f *frameLoop) resetImages(ctx context.Context, stateBuilder *stateBuilder)
 	imgPrimer := newImagePrimer(stateBuilder)
 	defer imgPrimer.Free()
 
+	for toCreate := range f.imageToCreate {
+		log.D(ctx, "Recreate image %v which was destroyed during loop.", toCreate)
+		image := GetState(f.loopStartState).Images().Get(toCreate)
+		stateBuilder.createImage(image, f.loopStartState, toCreate)
+		// For image creation, the associated image views changes are handled in the restore step below.
+	}
+
```

Comment:
here. (see above)

Reply:
I saw you removed the code above, but you were supposed to move it down here! And also add the imageView to imageViewToCreate. Am I just reading the diff incorrectly?

Reply:
Sorry, my bad. I see it now!!!